app/assets/stylesheets/comments.scss

```diff
@@ -22,15 +24,18 @@
     .media { margin: 5px; }
   }
 
+.comments > .comment {
+  padding: 10px 0px 10px 0px;
+  margin: 4px 10px 0px 10px;
```

Comment:
- Shorthands of length `4` are not allowed. Value was `4px 10px 0px 10px`
- Shorthand form for property `margin` should be written more concisely as `4px 10px 0px` instead of `4px 10px 0px 10px`
- `0px` should be written without units as `0`